pymoab-visit-py2-18.04

```diff
@@ -0,0 +1,28 @@
+FROM svalinn/pymoab-py2-18.04:latest
+
+ENV HOME /root
+
+RUN apt-get install -y --fix-missing \
+    wget \
+    cpio \
+    libpcre3-dev \
+    libgl1-mesa-glx \
+    libgl1-mesa-dev \
+    libsm6 \
+    libxt6 \
+    libglu1-mesa
+
+RUN pip install xmldiff
```

Comment:
maybe group all package install in a single line ?

Reply:
Is there a best practice on whether to separate or group pip installs w/ apt installs? Seems like they should be separated to me.

Reply:
I don't know, 

I don't feel like this is very important/sensitive so it should be regrouped into a single line to improve efficiency.